src/renderer/rendered_layer.rs

```diff
@@ -197,10 +197,10 @@ pub fn group_windows(
         .collect_vec()
 }
 
-fn build_silhouette(regions: &[PixelRect<f32>]) -> (Path, Rect) {
+fn build_silhouette(regions: &[PixelRect<f32>], settings: &RendererSettings, grid_scale: GridScale) -> (Path, Rect) {
     let silhouette = regions
         .iter()
-        .map(|r| Path::rect(to_skia_rect(r), None))
+        .map(|r| rect_to_round_rect_path(&to_skia_rect(r), settings, grid_scale))
```

Comment:
I'm worried about this change when there are multiple floating windows together, like in some of the telescope layouts for example. This could leave gaps in the middle of the silhouette. 

So, I think the rounding only should be done on the outside.

Reply:
But I need it to work this way, because I like how different parts of the telescope look like separate areas, like this:
<img width="1278" alt="image" src="https://github.com/user-attachments/assets/1020e02f-3a91-4de4-87ac-6d4be143ba31">

Maybe a configuration option could be added to control this effect.

Reply:
Yeah, I submitted new commit to fix this issue.

Reply:
But there’s another problem here: when a floating window has two rectangles that are nested, two shadows appear(eg. `folke/noice.nvim`'s cmdline popup):
<img width="716" alt="image" src="https://github.com/user-attachments/assets/e5f055bb-47d3-4c8e-9df1-fe05fc3e4270">

However, this should be a new issue.

Reply:
I believe this is a known issue. Theres no standard way currently to tell that two floating windows should exist at the same z level. On most guis this is rendered flat so its not a problem. We don't have that luxury

Reply:
I just tested it, that does not happen to me with the latest version of the plugin.